optaplanner-core/src/main/java/org/optaplanner/core/impl/constructionheuristic/DefaultConstructionHeuristicPhase.java

```diff
@@ -36,6 +38,13 @@
     protected EntityPlacer entityPlacer;
     protected ConstructionHeuristicDecider decider;
 
+    protected final boolean skipBestSolutionCloningInSteps = true;
```

Comment:
This is currently a constant. If it was the intention it should be renamed to upper case.

Since there is some decision logic based on this field, perhaps there should be a constructor parameter for it? Or is this planned for a future iteration?